test/gurka/test_costing_with_traffic.cc

```diff
@@ -0,0 +1,248 @@
+#include "gurka.h"
+#include "test.h"
+#include <gtest/gtest.h>
```

Comment:
This is the test that is for porting the logic from autocost to the other costing models.  Tests with and without traffic.  

https://github.com/valhalla/valhalla/pull/2944/files#r596212523

Reply:
also @mandeepsandhu thank you for the help with the tests.